src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -0,0 +1,724 @@
+    const timer = setInterval(() => {
+      setReportStalenessTimer((reportStalenessTimer) => reportStalenessTimer + 1);
+    }, interval);
+    return () => clearInterval(timer);
+  }, [
+    setReportStalenessTimer,
+    setReportStalenessTimerUnits,
+    reportTime,
+    reportStalenessTimer,
+    usingArchivedReport,
+    usingCachedReport,
+  ]);
+
+  React.useEffect(() => {
+    setFilteredCategorizedEvaluation(
+      filterAutomatedAnalysis(
+        categorizedEvaluation,
+        targetAutomatedAnalysisFilters,
+        targetAutomatedAnalysisGlobalFilters,
+        showNAScores
+      )
+    );
+  }, [
+    categorizedEvaluation,
+    targetAutomatedAnalysisFilters,
+    targetAutomatedAnalysisGlobalFilters,
+    showNAScores,
+    filterAutomatedAnalysis,
+    setFilteredCategorizedEvaluation,
+  ]);
+
+  const onCardExpand = React.useCallback(() => {
+    setIsCardExpanded(!isCardExpanded);
+  }, [setIsCardExpanded, isCardExpanded]);
+
+  const handleNAScoreChange = React.useCallback(
+    (checked: boolean) => {
+      setShowNAScores(checked);
+    },
+    [setShowNAScores]
+  );
+
+  const filteredCategorizedLabels = React.useMemo(() => {
+    return (
+      <Grid>
+        {filteredCategorizedEvaluation
+          .filter(([_, evaluations]) => evaluations.length > 0)
+          .map(([topic, evaluations]) => {
+            return (
+              <GridItem className="automated-analysis-grid-item" span={3} key={`gridItem-${topic}`}>
+                <LabelGroup categoryName={topic} isVertical numLabels={3} isCompact key={`topic-${topic}`}>
+                  {evaluations.map((evaluation) => {
+                    return (
+                      <ClickableAutomatedAnalysisLabel
+                        label={evaluation}
+                        isSelected={false}
+                        key={clickableAutomatedAnalysisKey}
+                      />
+                    );
+                  })}
+                </LabelGroup>
+              </GridItem>
+            );
+          })}
+      </Grid>
+    );
+  }, [filteredCategorizedEvaluation]);
+
+  const clearCacheStartRecording = React.useCallback(() => {
+    addSubscription(
+      context.target.target().subscribe((target) => {
+        context.reports.deleteCachedAnalysisReport(target.connectUrl);
+        startProfilingRecording();
+      })
+    );
+  }, [addSubscription, context.target, context.reports, startProfilingRecording]);
+
+  const clearAnalysis = React.useCallback(() => {
+    setIsLoading(true);
+    if (usingCachedReport) {
+      context.reports.deleteCachedAnalysisReport(targetConnectURL);
+    } else if (usingArchivedReport) {
+      // do nothing, we don't want to delete unrelated archived reports
+      setIsLoading(false);
+      return;
+    } else {
+      addSubscription(
+        context.api.deleteRecording('automated-analysis').subscribe(() => {
+          context.reports.deleteCachedAnalysisReport(targetConnectURL);
```

Comment:
Should call `handleStateErrors` in here after recording operation finishes. We might need to handle error here too if failing to delete the recording.